### ISSUE TITLE
When exporting to python, put the tool label in comment

### ISSUE THREAD

**Issue by simonaubertbd** [amphi-ai/amphi-etl#79]
Hello,

Here a simple workflow
![image](https://github.com/user-attachments/assets/5f066a4c-4b5b-42b6-92f6-c6ae2d2e48c7)

and here the python code
```python
# Source code generated by Amphi
# Date: 2024-08-13 15:27:48
# Additional dependencies: psycopg2-binary
import pandas as pd
import sqlalchemy
import psycopg2
import os


# Connection constants for Postgres
POSTGRES_HOST = "localhost"
POSTGRES_PORT = "5433"
POSTGRES_DATABASE_NAME = "postgres"
POSTGRES_USERNAME = "postgres"
POSTGRES_PASSWORD = "*************"
POSTGRES_SCHEMA = ""



# Connect to the PostgreSQL database
postgresInput1_Engine = sqlalchemy.create_engine(f"postgresql://{POSTGRES_USERNAME}:{POSTGRES_PASSWORD}@{POSTGRES_HOST}:{POSTGRES_PORT}/{POSTGRES_DATABASE_NAME}")

# Execute SQL statement
try:
    with postgresInput1_Engine.connect() as conn:
        postgresInput1 = pd.read_sql(
            """SELECT * FROM bd_market.articles""",
            con=conn.connection
        ).convert_dtypes()
finally:
    postgresInput1_Engine.dispose()

print(postgresInput1.head(100))
``` 

This would be pretty nice to get my tool labels (such as Postgres Input test article ) in the generated python code.

Best regards,

Simon

**Comment by tgourdel** [comment]
Great suggestion, thanks Simon!

**Comment by tgourdel** [comment]
Thanks Simon, released in 0.8.2

**Comment by simonaubertbd** [comment]
@tgourdel  Thanks !

Here an example :
![image](https://github.com/user-attachments/assets/4b1ab189-f15a-4800-b3e9-d8b242b2f3e8)

So, it works for the first but not the the second, I will open an issue.

Best regards,

Simon
